Group-6.cpp

```diff
@@ -0,0 +1,126 @@
+#include <iostream>
+#include <stdlib.h>
+#include <string.h>
+using namespace std;
+
```

Comment:
Initialize variables to the sizes of the arrays instead of using constant variables